packages/es-indexer/src/indexers/TrackIndexer.ts

```diff
@@ -99,7 +99,7 @@ export class TrackIndexer extends BaseIndexer<TrackDoc> {
         is not null then true
         else false
       end as purchaseable,
-      (tracks.download->>'is_downloadable')::boolean as downloadable,
+      tracks.is_downloadable as downloadable,
```

Comment:
@stereosteve for visibility. in case some other change or re-indexing is needed somewhere

Reply:
Should be fine without needing to build a new index.